How-To/Add-columns-to-open-data-assets.md

```diff
@@ -0,0 +1,23 @@
+# Add Columns to Open Data Assets
+
+The data team maintains a motley collection of [open data assets](https://datacatalog.cookcountyil.gov/browse?tags=cook+county+assessor). The [Athena views](https://ccao-data.github.io/data-architecture/#!/overview/ccao_data_athena) that feed these assets are updated regularly with new features, and it's important to maintain parity.
+
+1. [Log in to](https://datacatalog.cookcountyil.gov/login) the open data portal using the Data team's credentials.
```

Comment:
```suggestion
1. [Log in to](https://datacatalog.cookcountyil.gov/login) the Open Data Portal using the Data Department's credentials.
```

I _think_ this is correct? Technically Open Data Portal is a proper noun? Or maybe it's just a noun, idk.